tests/unit_tests/DescriptorSetAdd_test.cc

```diff
@@ -218,6 +218,116 @@ TEST(Descriptors_Add, add_flatl2_100d_2add) {
   delete[] xb;
 }
 
+// HNSW Tests
+TEST(Descriptors_Add, add_hnswflatl2_100d) {
+  int d = 100;
+  int nb = 10000;
+  float *xb = generate_desc_linear_increase(d, nb);
+
+  std::string index_filename = "dbs/add_hnswflatl2_100d";
+  VCL::DescriptorSet index(index_filename, unsigned(d), VCL::FaissHNSWFlat);
+
+  std::vector<long> classes(nb);
+
+  for (auto &str : classes) {
+    str = 1;
+  }
+
+  index.add(xb, nb, classes);
+
+  std::vector<float> distances;
+  std::vector<long> desc_ids;
+  index.search(xb, 1, 4, desc_ids, distances);
+
+  int exp = 0;
+  // std::cout << "DescriptorSet: " << std::endl;
+  for (auto &desc : desc_ids) {
+    // std::cout << desc << " ";
+    EXPECT_EQ(desc, exp++);
+  }
+  // std::cout << std::endl;
+
+  // std::cout << "Distances: " << std::endl;
+  float results[] = {float(std::pow(0, 2) * d), float(std::pow(1, 2) * d),
+                     float(std::pow(2, 2) * d), float(std::pow(3, 2) * d)};
+  for (int i = 0; i < 4; ++i) {
+    // std::cout << distances[i] <<  " ";
+    EXPECT_EQ(distances[i], results[i]);
+  }
+  // std::cout << std::endl;
```

Comment:
Please remove comments used for debugging
```suggestion
  for (auto &desc : desc_ids) {
    EXPECT_EQ(desc, exp++);
  }

  float results[] = {float(std::pow(0, 2) * d), float(std::pow(1, 2) * d),
                     float(std::pow(2, 2) * d), float(std::pow(3, 2) * d)};
  for (int i = 0; i < 4; ++i) {
    EXPECT_EQ(distances[i], results[i]);
  }
```

Reply:
done